tst/unit/kokkos_abstraction.cpp

```diff
@@ -622,8 +622,9 @@ struct SmallNLongTBufferPack {
   template <typename TimeType>
   static void test_time(const TimeType time_default, const TimeType time_spaces,
                         const int nspaces) {
```

Comment:
So we don't need `nspaces` anymore? In that case we should do:
```suggestion
                        const int /*nspaces*/) {
```

Reply:
Will handle that in a separate PR not not trigger CI again.

Reply:
Curious, is this because of `-Wunused-parameter` ? Why not remove the `const int` too?